README.md

```diff
@@ -244,10 +247,53 @@ Both `pred_path` (if `metric = miou`) and `gt_path` must be json files where eac
+
+<a name="regression_pathology"></a>
+## Run regressions on pathology features
+We provide a script to run a simple linear regression with the evaluation metric (IoU or hit rate) as the dependent variable (to understand the relationship between the geometric features of a pathology and saliency method localization performance). Each regression uses one of the above four geometric features as a single independent variable.
+
+```
+(chexlocalize) > python regression.py [FLAGS]
+```
+
+**Required flags**
+* `--features_dir`: Path to directory that holds four csv files: `area_ratio.csv`, `elongation.csv`, `num_instances.csv`, and `rec_area_ratio.csv`. These four files are the output of `compute_pathology_features.py`.
+* `--pred_miou_results`: path to csv file with saliency method IoU results for each CXR and each pathology. This is the output of `eval.py` called `miou_results_per_cxr.csv`.
+* `--pred_hitrate_results`: path to csv file with saliency method hit/miss results for each CXR and each pathology. This is the output of `eval.py` called `hitrate_results_per_cxr.csv`.
+
+**Optional flags**
+* `--evalute_hb`: Default is `False`. If true, evaluate human benchmark in addition to saliency method. If `True`, the flags `hb_miou_results` and `hb_hitrate_results` (below) are also required. If `True`, additional regressions will be run using the difference between the evaluation metrics of the saliency method pipeline and the human benchmark as the dependent variable (to understand the relationship between the geometric features of a pathology and the gap in localization performance between the saliency method pipeline and the human benchmark).
+* `--hb_miou_results`: Path to csv file with human benchmark IoU results for each CXR and each pathology. This is the output of `eval.py` called `miou_results_per_cxr.csv`.
+* `--hb_hitrate_results`: Path to csv file with human benchmark hit/miss results for each CXR and each pathology. TODO: This is the output of `eval.py` called `hitrate_results_per_cxr.csv`.
+* `--save_dir`: Where to save regression results. Default is current directory. If `evaluate_hb` is `True`, four files will be saved: `regression_pred_miou.csv`, `regression_pred_hitrate.csv`, `regression_miou_diff.csv`, `regression_hitrate_diff.csv`. If `evaluate_hb` is `False`, only two files will be saved: `regression_pred_miou.csv`, `regression_pred_hitrate.csv`.
+
+In [our paper](https://www.medrxiv.org/content/10.1101/2021.02.28.21252634v3), only the true positive slice was included in each regression (see Table 2). Each feature is normalized using z-score normalization (TODO: is this true and not min max??) and the regression coefficient can be interpreted as the effect of that geometric feature on the evaluation metric at hand.
```

Comment:
@alexaaag in our paper, we say that we use z-score normalization, but is this true? I think the script uses min-max?

Reply:
Yes I left a comment in the previous email to change it to min-max normalization. Can you revise the paper? I will take one more read after you are done.